cypress/integration/appTest.js

```diff
@@ -23,4 +23,12 @@ describe('App Home Page', () => {
     .contains("Raleigh").click().url().should('include', 'http://localhost:3000/3');
   });
 
+
+  it('User should be able to type in the search bar and see the cities filtered on the page', () => {
+    cy.get('input[type=text]').type('D')
+    cy.get('.cities-container').children().should('have.length', 1)
+
+    cy.get('input[type=text]').type('e').clear()
+    cy.get('.cities-container').children().should('have.length', 4)
+  })
```

Comment:
I need to study some more Cypress!  Haven't seen any testing around typing into input fields, so I'm thrilled that you were able to add this piece in!